packages/account/src/Components/file-uploader/file-uploader.jsx

```diff
@@ -104,11 +104,11 @@ const FileUploader = React.forwardRef(({ onFileDrop, getSocket }, ref) => {
                 value={document_file.files}
             />
             {(document_file.files.length > 0 || !!document_file.error_message) && (
-                <div className='account-poa__upload-remove-btn-container'>
+                <div className='account__file-uploader-remove-btn-container'>
                     <Icon
```

Comment:
here and further in the code "-poa__upload" changed to "__file-uploader". existing file-uploader is supposed to be reusable component without pointing exact feature classes.